### PR TITLE
Code: Fixes VC warnings

### PR DESCRIPTION
**Pull request by am11**
//cc @mgreter

### CODE REVIEW

**Comment by xzyfer** [comment]
Gosh VS is so picky ;)

**Comment by xzyfer** [comment]
Master fails to compile with clang now. This appears to fix this.

```
fatal error: too many errors emitted, stopping now [-ferror-limit=]
```

File: ast.hpp [hunk 1]
```diff
@@ -1523,7 +1523,7 @@ namespace Sass {
 
       if (with)
       {
-        if (!l || !l->length()) return str.compare("rule");
+        if (!l || !l->length() != 0) return str.compare("rule") != 0;
```
Comment (xzyfer) [inline ast.hpp]: Shoudl be `if (!l || l->length() != 0)`

Comment (am11) [inline ast.hpp]: Oops. TravisCI was passing, I didn't ran tests locally. :p
Thanks for fixing it! :+1:

**Pull request merged**